databases/week1/design.sql

```diff
@@ -0,0 +1,10 @@
+SELECT COUNT(*) AS task_count FROM tasks;
+SELECT COUNT(*) AS task_count FROM tasks WHERE due_date IS NULL ;
+SELECT * FROM tasks WHERE status = 'done';
+SELECT * FROM tasks WHERE status != 'done';
+SELECT * FROM tasks ORDER BY created_date DESC;
+SELECT * FROM tasks ORDER BY created_date DESC LIMIT 1;
+SELECT title, due_date FROM tasks WHERE title LIKE '%database%' OR description LIKE '%database%';
+SELECT title, CASE status WHEN 'done' THEN 'Done' WHEN 'in_progress' THEN 'In Progress' ELSE 'Not Started' END AS status_text FROM tasks;
```

Comment:
Here you need to use [JOIN](https://www.w3schools.com/mysql/mysql_join.asp) to combine two tables and  retrieve the data from both of them. You would match the tables based on status id and, thus, easily get required data. ☺️ 

The reason for using Join: Imagine that you have thousands of records in your database, you definitely wouldn't want to think of every possible case for status name or any other criteria. That's why we have joins to help us out 😉